sequencer/src/api/options.rs

```diff
@@ -356,7 +356,7 @@ impl Options {
             // Initialize merklized state module for fee merkle tree
             app.register_module(
                 "fee-state",
-                endpoints::merklized_state::<N, P, _, FeeMerkleTree, _, 256>()?,
+                endpoints::get_balance::<_, SequencerApiVersion>()?,
```

Comment:
It's probably fine but we are replacing and API so some endpoints will be removed. I guess nobody needs them but then I'm wondering why we had it in the first place 🤔

Reply:
The new API should have all the endpoints of the original one, because the new endpoint is added as an extension to the hotshot-query-service `merklized_state` API